src/MobileMapping/Sensor.js

```diff
@@ -87,6 +87,7 @@ Sensor.prototype.getMatOrientationCapteur = function getMatOrientationCapteur()
         case 2:
             return ori2;
         case 3:
+        default:
```

Comment:
It depends on how normal the "this.orientation is not 0, 1, 2 or 3" is... I honestly don't know here so I leave that to your fine judgment :-)